client/swagger.yaml

```diff
@@ -395,7 +385,7 @@ definitions:
 
   Error:
     properties:
-      fault_message:
+      faultMessage:
```

Comment:
We actually don't want this change. `fault_message` is correct, and there's an open PR to get it fixed in firecracker. See https://github.com/firecracker-microvm/firecracker/pull/713#discussion_r238536099